### ISSUE TITLE
Gogoanime: Streaming links not working

### ISSUE THREAD

**Issue by carlosesteven** [consumet/api.consumet.org#480]
### Describe the bug

The endpoint stopped working a few hours ago.

### Steps to reproduce

1. Fetch: https://api.consumet.org/anime/gogoanime/watch/spy-x-family-episode-1

### Expected behavior

```
{
  "headers": {
      "Referer": "string",
  },
  "sources": [
      {
      "url": "string",
      "quality": "string",
      "isM3U8": true
      }
  ]
}
```

Based on: https://docs.consumet.org/rest-api/Anime/gogoanime/get-anime-episode-streaming-links

### Actual behavior

`{"message":{}}`

### Additional context

- This happens with all the episode links.
- The provider updated the domain from: https://gogoanimehd.to to https://gogoanimehd.io.

**Comment by dopeCape** [comment]
@carlosesteven  just made a pr regarding this issue,
https://github.com/dopeCape/consumet.ts
you can use this fork of their scrapper until the pr is merged.

**Comment by Anuragkumarsah** [comment]
> @carlosesteven just made a pr regarding this issue, https://github.com/dopeCape/consumet.ts you can use this fork of their scrapper until the pr is merged.

I tried installing this fork using (npm install https://github.com/dopeCape/consumet.ts)  but still I am getting the same error Episode not found. Can you please help me out 🙏

**Comment by carlosesteven** [comment]
> > @carlosesteven just made a pr regarding this issue, https://github.com/dopeCape/consumet.ts you can use this fork of their scrapper until the pr is merged.
> 
> I tried installing this fork using (npm install https://github.com/dopeCape/consumet.ts) but still I am getting the same error Episode not found. Can you please help me out 🙏

@Anuragkumarsah Update this line: https://github.com/carlosesteven/api.consumet.org/blob/cc625dcdd9b3e8c856702c3c2598ec68214587ca/package.json#L26 

Then run again: npm install

**Comment by riimuru** [comment]
I have merged the changes on [pr #376](https://github.com/consumet/consumet.ts/pull/376) should be fixed now.
